tests/test_responses.py

```diff
@@ -330,7 +330,8 @@ def test_empty_response(test_client_factory):
     app = Response()
     client: TestClient = test_client_factory(app)
     response = client.get("/")
-    assert response.headers["content-length"] == "0"
+    assert response.status_code == 204
+    assert "content-length" not in response.headers
```

Comment:
@Kludex Does that `content-length` make sense? Or should we include `content-length` in empty response with 204 status code?

Reply:
Yes. https://datatracker.ietf.org/doc/html/rfc7230#section-3.3.2

>  A server MUST NOT send a Content-Length header field in any response
   with a status code of 1xx (Informational) or 204 (No Content).  A
   server MUST NOT send a Content-Length header field in any 2xx
   (Successful) response to a CONNECT request (Section 4.3.6 of
   [RFC7231]).

Reply:
Great thanks, just wanted to double check since you were recently in this area.

Reply:
It's good to have a reference everywhere, so we can come back in the future and see the reason. Makes things faster anyway :P